app/views/users/registrations/new.html.erb

```diff
@@ -24,6 +24,13 @@
   </div>
 
   <div class="field">
+    <%= f.fields_for :picture, resource.picture do |picture_fields| %>
+      <%= picture_fields.label :profile_picture %><br />
+      <%= picture_fields.file_field :image,accept: 'image/png,image/gif,image/jpeg'%>
```

Comment:
@charusat09 Yes, we have already handle validation in model but I also handle front end validation so one can play with it.